wolfssl/wolfcrypt/settings.h

```diff
@@ -1318,6 +1325,36 @@ extern void uITRON4_free(void *p) ;
     #define GCM_TABLE
 #endif
 
+#if defined(WOLFSSL_MAXQ1065) || defined(WOLFSSL_MAXQ108X)
+
+    #define MAXQ10XX_MODULE_INIT
+
+    #define HAVE_PK_CALLBACKS
+    #define WOLFSSL_STATIC_PSK
+
+    #define WOLFSSL_USER_MUTEX
+    #ifdef WOLFSSL_USER_MUTEX
+        #define WOLFSSL_CRYPT_HW_MUTEX 1
+        #define MAXQ10XX_MUTEX
+    #endif
+
+    #define WOLFSSL_MAXQ10XX_CRYPTO
+    #define WOLFSSL_MAXQ10XX_TLS
+
+    #if defined(WOLFSSL_MAXQ10XX_TLS) && !defined(NO_WOLFSSL_SERVER)
+        #error "The MAXQ10XX does not support server side functionalities."
```

Comment:
Please add code comment for why only client side is supported.

Reply:
TODO

Reply:
Fixed with a083dae27abcb69198aedbb9da3ef27f119ac9a9 .